eng/native/naming.props

```diff
@@ -9,7 +9,7 @@
         <SymbolFileExtension>.pdb</SymbolFileExtension>
       </PropertyGroup>
     </When>
-    <When Condition=" '$(_runtimeOSFamily)' == 'osx' OR $(RuntimeIdentifier.StartsWith('osx')) ">
+    <When Condition=" '$(_runtimeOSFamily)' == 'osx' OR $(RuntimeIdentifier.StartsWith('osx')) OR '$(_runtimeOSFamily)' == 'ios' OR $(RuntimeIdentifier.StartsWith('ios'))">
```

Comment:
Use lowercase conditionals (or/and). Some files haven't been cleaned up, but we try and use lowercase in MSBuild conditions.